site/src/js/index.js

```diff
@@ -1,103 +1,106 @@
-/* global Selectr */
+/* global window, document, Selectr */
```

Comment:
The environment should be specified in the XO config

Reply:
As of https://github.com/xojs/xo/pull/279 I think we can keep these globals in this file from now.

Here's the config for reference
```
  "xo": {
    "overrides": [
      {
        "files": "src/js/**/*.js",
        "globals": ["document", "window"]
      }
    ]
  },
```

Reply:
Instead of `globals` you can use `envs: ["browser"]`